package ExceptionHandling;

public class TryCatchExample1 {
public static void main(String[] args) {
	System.out.println("Hi");
	try {
		System.out.println(10/0);
	}
	catch(ArithmeticException a) {
		System.out.println("incorrect");
	}

	System.out.println("Bye");
}
}
